middleware.ts

```diff
@@ -11,16 +11,17 @@ export async function middleware(req: NextRequest) {
     data: { session },
   } = await supabase.auth.getSession();
 
+  const redirectUrl = req.nextUrl.clone();
```

Comment:
Why move this up?

Reply:
Forgot to revert back after removing redirect check made